src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -334,54 +317,28 @@
             "The provided %s is not acceptable to TEAMMATES as it cannot be empty.";
     
     /**
-     * 
-     * @param fieldType
-     *            The field type. e.g., FieldType.E_MAIL
-     * @param value
-     *            The value of the field. e.g., "david@yahoo.com"
-     * @return A string explaining reasons why the value is not acceptable and
-     *         what are the acceptable values. Returns an empty string "" if the
-     *         value is acceptable
-     */
-    public String getInvalidityInfo(FieldType fieldType, Object value) {
-        return getInvalidityInfo(fieldType, "", value);
+     * Checks if {@code emailContent} is not null and not empty
+     * @param emailContent
+     * @return An explanation of why the {@code emailContent} is not acceptable.
+     *         Returns an empty string if the {@code emailContent} is acceptable.
+     */
+    public String getInvalidityInfoForEmailContent(Text emailContent) {
+        Assumption.assertTrue("Non-null value expected", emailContent != null);
```

Comment:
Although not directly related to your PR, perhaps it might be good to (not necessarily in this PR) replace this line with
`Assumption.assertNotNull(ERROR_NULL_PARAMETER, emailContent);` as well :)